src/main/scala/uk/gov/nationalarchives/aws/utils/Clients.scala

```diff
@@ -40,5 +42,14 @@ object Clients {
       .build()
   }
 
+  def ecr: EcrAsyncClient = {
+    val httpClient = NettyNioAsyncHttpClient.builder.build
+    EcrAsyncClient.builder
+      .region(Region.EU_WEST_2)
+      .endpointOverride(URI.create(configFactory.getString("ecr.endpoint")))
+      .httpClient(httpClient)
+      .build
+  }
+
```

Comment:
Unnecessary line breaks

Reply:
I was doing so well with the line breaks up to this point :smile: